### PR TITLE
drake comm call post

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

**Comment by stefaniebutland** [comment]
@wlandau Please check this draft announcement for your Community Call and suggest any edits, including for bio text. I have used the title and abstract exactly as you provided. 

Note that in bio I'll remove the extra photo - it's a placeholder till I figure out how to align the photo with text to the right using Hugo shortcodes.

If possible, I'd like to publish Thursday, Aug 8, but can wait if needed.

File: content/blog/2019-08-08-commcall-sep2019.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- The [drake](https://github.com/ropensci/drake) R package
+- drake [User Manual](https://ropenscilabs.github.io/drake-manual/)
+- Learn drake [workshop](https://github.com/wlandau/learndrake)
+
+#### Speaker
+
+<img src="/img/blog-images/2019-08-08-commcall-sep2019/will-landau.jpg" alt="headshot of Will Landau" style="margin: 0px 20px; width: 200px;" align="left">
+_**Will Landau** is ...._  
+
+{{< figure src="/img/blog-images/2019-08-08-commcall-sep2019/will-landau.jpg" alt="headshot of Will Landau" width="200" align="left">}} _**Will Landau** does statistics and writes software in the life sciences industry. He is a Research Statistician at Eli Lilly and Company. Will has a PhD in Statistics._
```
Comment (wlandau) [inline content/blog/2019-08-08-commcall-sep2019.md]: How about this:

**Will Landau** is a Research Scientist at Eli Lilly and Company, where he develops  methods and software for statisticians. Prior to joining Lilly, he earned his PhD in Statistics from Iowa State University.

File: content/blog/2019-08-08-commcall-sep2019.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#### Resources
+
+- The [drake](https://github.com/ropensci/drake) R package
+- drake [User Manual](https://ropenscilabs.github.io/drake-manual/)
+- Learn drake [workshop](https://github.com/wlandau/learndrake)
+
+#### Speaker
+
+<img src="/img/blog-images/2019-08-08-commcall-sep2019/will-landau.jpg" alt="headshot of Will Landau" style="margin: 0px 20px; width: 200px;" align="left">
+_**Will Landau** is ...._
```
Comment (wlandau) [inline content/blog/2019-08-08-commcall-sep2019.md]: As you said, one of the photos is a duplicate, as is the caption.

**Comment by wlandau** [comment]
Looks great, @stefaniebutland! My comments are minor.

**Comment by stefaniebutland** [comment]
Thanks @wlandau. Have updated your bio using your text. Will fix duplicate photo and text.

I'll tag you in a tweet on Thursday :-)

**Comment by wlandau** [comment]
Awesome! Thank you again for this opportunity!

**Pull request merged**